src/main/java/org/libtorrent4j/AnnounceFlags.java

```diff
@@ -0,0 +1,27 @@
+package org.libtorrent4j;
+
+/**
+ * @author aldenml
+ */
+public final class AnnounceFlags {
+
+    private AnnounceFlags() {
+    }
+
+    /**
+     * Announce to DHT as a seed.
+     */
+    public static final byte SEED = 1; // 1 << 0
+
+    /**
+     * Announce to DHT with the implied-port flag set. This tells the network to use
+     * your source UDP port as your listen port, rather than the one specified in
+     * the message. This may improve the chances of traversing NATs when using uTP.
+     */
+    public static final byte IMPLIED_PORT = (1 << 1);
+
+    /**
+     * Specify the port number for the SSL listen socket in the DHT announce.
+     */
+    public static final byte SSL_TORRENT = 1 << 2;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5839370)